RSFormView/Classes/Models/FormField.swift

```diff
@@ -78,6 +78,9 @@ open class FormField {
   /// The maximum selectable date for *.date* FormFields
   public var maximumDate: Date?
   
+  /// Source data for *.picker* FormFields
+  public var options: [String]?
```

Comment:
I think this class is getting too many responsibilities. We should refactor this to a more OO solution where we have FormField as a parent class and then PickerField, TextField, etc as subclasses. This is more like a V2 refactor, but just to raise the issue. @germanStabile WDYT?

Reply:
true! we will look into creating a new card so we can take care of this!!

Reply:
[Trello Card to tackle this](https://trello.com/c/TjB6391q/18-refactor-formfield-to-a-oo-solution-where-formfield-will-be-the-parent-class-and-different-fields-the-subclasses)